snap/snapcraft.yaml

```diff
@@ -233,29 +229,18 @@ parts:
       - libxcb-dri3-0:amd64
       - libxcb-dri3-0:i386
       - libpci3
-      - libvkd3d1:i386
-      - libvkd3d1:amd64
       - libvulkan1:i386
       - libvulkan1:amd64
```

Comment:
Are these still required?

Reply:
`libvulkan1` I mean

Reply:
Seems to work without `libvulkan1`, not sure if there are things that may require it.

Reply:
Hmm, should we maybe put `libvulkan1` in `gaming-graphics`? That's the vulkan loader library.

Reply:
In general, `gaming-graphics` doesn't quite fit together for me. I'm not sure `libvk3d1` since that's Direct3D->Vulkan translator, which is the odd-one-out since everything else is drivers. It feels to me like either that (and `vulkan-tools` and `mesa-tools`) should be removed, *or*, things like `libgl1` and `libvulkan1` should be in there.

Reply:
I do think these could be interesting in the gaming-graphics snap, even though they aren't drivers or coming from those PPAs.  At some point we could choose to build parts of those as parts and introduce various risk levels which would align with the content snap.

Reply:
That's a good suggestion, just updated with a debug tools part.

I'll open a PR for the gaming-graphics-core22 snap adding `libvulkan1` and if that ends up getting merged we can remove it here as well.